src/core/sys/posix/arpa/inet.d

```diff
@@ -199,23 +199,4 @@ NOTE: The following must must be defined in core.sys.posix.arpa.inet to break
 INET6_ADDRSTRLEN // from core.sys.posix.netinet.in_
 */
 
-version( linux )
-{
-    enum INET6_ADDRSTRLEN = 46;
-}
-else version( OSX )
-{
-    enum INET6_ADDRSTRLEN = 46;
-}
-else version( FreeBSD )
-{
-    enum INET6_ADDRSTRLEN = 46;
-}
-else version( Solaris )
-{
-    enum INET6_ADDRSTRLEN = 46;
-}
-else version( Android )
-{
-    enum INET6_ADDRSTRLEN = 46;
-}
+enum INET6_ADDRSTRLEN = 46;
```

Comment:
[Defined to be 46 in the Posix standard](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/netinet_in.h.html).